docs/developer-guides/setting-up-token-management-partner-chains.md

```diff
@@ -0,0 +1,408 @@
+# Setting up token management for a partner chain
+
+This guide shows you how to set up and manage a token management contract on the Cardano blockchain, enabling proper observability on partner chains. 
+This guide presents the step-by-step process involving interactions with both the Cardano and the partner chain blockchain.
```

Comment:
I recommend that we add a heading "Overview" above these two statements. Also, it's very important that these statements accurately describe the scope and purpose of this document. @LGLO and @alexromanov, can you suggest any changes here to make sure we have an accurate overview of this doc?

Reply:
- We need to mention that "Rewards schemes for token release is the responsibility of the Chain Builder"
- We can split the document into few logical parts: initialization, usage, configuration
- Initialization will include creating native token on Cardano, writing V-function and running `init-reserve-management` and `reserve-create` commands at the `pc-contracts-cli` side
- Usage part will include `reserve-release-funds` and `releae-handover` commands
- Configuration part will include `reserve-deposit` and `reserve-update-settings` commands

Reply:
Thanks @alexromanov, that's very helpful info!